src/utils/dateUtils.ts

```diff
@@ -0,0 +1,45 @@
+export type Kvartal = 1 | 2 | 3 | 4;
+export type KvartalÅrstall = { kvartal: number, årstall: number }
+
+export const toKvartalPeriode = (fra: { kvartal: Kvartal, årstall: number }, til: { kvartal: Kvartal, årstall: number }) => {
+  return `${kvartalStartDato(fra.kvartal)} ${fra.årstall} til ${kvartalSluttDato(til.kvartal)} ${til.årstall}`
+}
+
+const kvartalStartDato = (kvartal: Kvartal) => {
+  switch (kvartal) {
+    case 1:
+      return "1. januar.";
+    case 2:
+      return "1. april.";
+    case 3:
+      return "1. juli.";
+    case 4:
+      return "1. oktober.";
+  }
+}
+
+const kvartalSluttDato = (kvartal: Kvartal) => {
```

Comment:
Disse datoene er ikke faste hvert år dessverre, men vi kan ta i bruk det nye publiseringsdato-apiet for å hente ut riktige datoer